CMakeLists.txt

```diff
@@ -6059,6 +6059,8 @@ set(ZIG_LIBC_FILES
     "include/x86_64-linux-musl/bits/stat.h"
     "include/x86_64-linux-musl/bits/syscall.h"
     "include/x86_64-linux-musl/bits/user.h"
+    "include/wasm32-freestanding-musl/bits/alltypes.h"
```

Comment:
Should we have created `libc/musl/arch/wasm32/bits/alltypes.h.in` instead?

Reply:
I'm not sure why we even have those .in files; it might be a mistake.

Reply:
Want to make an issue to follow that up?